README.md

```diff
@@ -79,70 +94,69 @@ and build the project:
-Once chat client is started, use `/add <name1>` to create a new connection and generate an invitation to send to your contact via any other communication channel (`<name1>` - is any name you want to use for that contact).
+This diagram shows how to connect and message a contact:
 
-Invitation has format `smp::<server>::<queue_id>::<rsa_public_key_for_this_queue_only>` - this needs to be shared with another party, via any other channel. This invitation can only be used once - even if this is intercepted, the attacker would not be able to use it to send you the messages via this queue once your contact confirms that the connection is established.
+<div align="center">
+  <img align="center" src="images/how-to-use-simplex.svg">
+</div>
 
-The party that received the invitation should use `/connect <name2> <invitation>` to accept the connection (`<name2>` is any name that the accepting party wants to use for you).
+Once you have started the chat, use `/add <name1>` to create a new connection and generate an invitation (`<name1>` - is any name you want to use for that contact). The add command will output an invitation. Send this invitation to your contact via any other channel.
 
-For example, if Alice and Bob want to chat, with Alice initiating, Alice would use [in her chat client]:
+The invitation has the format `smp::<server>::<queue_id>::<rsa_public_key_for_this_queue_only>`. The invitation can only be used once and even if this is intercepted, the attacker would not be able to use it to send you the messages via this queue once your contact confirms that the connection is established.
 
-```
-/add bob
-```
-
-And then send the generated invitation to Bob out-of-band. Bob then would use [in his chat client]:
+The contact who received the invitation should use `/connect <name2> <invitation>` to accept the connection (`<name2>` is any name that the accepting contact wants to use for you).
 
-```
-/connect alice <alice's invitation>
-```
+Once the contact has entered the `/connect` command, a connection is established and both parties are notified.
 
 They would then use `@<name> <message>` commands to send messages. One may also press Space or just start typing a message to send a message to the contact that was the last.
 
 If your contact is disconnected, restart the chat client - it may happen if you lose internet connection.
 
 Use `/help` in chat to see the list of available commands.
 
-## Accessing chat history
+### Access chat history
 
-SimpleX chat stores all your contacts and conversations in the local database file, making it private and portable by design, fully owned and controlled by you.
+SimpleX chat stores all your contacts and conversations in a local database file, making it private and portable by design, fully owned and controlled by you.
 
 You can search your chat history via SQLite database file:
 
 ```
 sqlite3 ~/.simplex/smp-chat.db
 ```
 
-Now you can query `messages` table, for example:
+You can query `messages` table:
```

Comment:
I'd keep the previous formulation because technically you can do and query whatever you want - not only messages table, but also other tables, as well as manually make data altering changes if you're careful enough - it wasn't said explicitly but at least implied by "for example"